factcast-server-grpc/src/main/java/org/factcast/server/grpc/FactCastSecurityConfiguration.java

```diff
@@ -102,22 +163,31 @@ DaoAuthenticationProvider daoAuthenticationProvider(UserDetailsService uds) {
 
     // security off
     @Bean
-    @ConditionalOnMissingBean(CredentialConfiguration.class)
-    UserDetailsService godModeUserDetailsService() {
-        log.warn(
-                "**** FactCast Security is disabled. This is discouraged for production environments. You have been warned. ****");
-        List<GrantedAuthority> fullAccess = AuthorityUtils
-                .commaSeparatedStringToAuthorityList(FactCastRole.READ + ","
-                        + FactCastRole.WRITE);
-        String DISABLED = "security_disabled";
-        return username -> new User(DISABLED, DISABLED, fullAccess);
+    @ConditionalOnMissingBean(FactCastAccessConfiguration.class)
+    UserDetailsService godModeUserDetailsService(
+            @org.springframework.beans.factory.annotation.Value("${insecure:#{false}}") Boolean insecureIsOk) {
+        if (insecureIsOk) {
+            log.warn(
+                    "**** FactCast Security is disabled. This is discouraged for production environments. You have been warned. ****");
+            return username -> new FactCastUser(FactCastAccount.GOD, "DISABLED");
+        }
+
+        log.error("**** FactCast Security is disabled. ****");
+        log.error("* If you really want to, you can run Factcast in such a configuration ");
+        log.error("* by adding a property '-Dinsecure' to your setup. However, it is");
```

Comment:
`-Dinsecure` doesn't work due to it being the "big" boolean. It's `null` then. It needs an explicit value assigned.